TodoList/AppDelegate.swift

```diff
@@ -1,46 +1,20 @@
-//
-//  AppDelegate.swift
```

Comment:
Are we deleting these auto-generated comments?

Reply:
I tend to do so. We can discuss it but I don't see the point in having them.

Reply:
It was just, so I could follow the same pattern.